Tests/SwerverTests/ResponseSpec/HttpResponseSpec.swift

```diff
@@ -5,48 +5,104 @@ import Foundation
 
 class HttpResponseSpec: QuickSpec {
     override func spec() {
-        describe("HttpResponse") {
-            var responseBuilder = ResponseBuilder()
+        describe("A Reponse object") {
+            func summonBuilder() -> HttpResponseBuilder {
```

Comment:
The name `summonBuilder` is a bit odd. Although also weird, the name `buildBuilder` is both accurate and consistent with your other tests.

Reply:
will change that